src/styles/product-card.scss

```diff
@@ -0,0 +1,74 @@
+.product-card {
+  max-width: 200px;
```

Comment:
Regarding @tiserett comment
Generally, he is right - but in this very case it is better to set the width for the card and let its elements fill its space
The problem with your recent solution is that you are limiting max width, not the min, so cards can shrink below 200px

I would suggest you putting back `width: 200px` here

Reply:
<img width="1007" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/1983879/88603598-d963-45ce-be38-29eddf351f9b">